packages/web/locales/en.json

```diff
@@ -1,41 +1,65 @@
 {
-  "language":"language",
-  "english":"english",
-  "german":"german",
-  "french":"french",
+  "language":"english",
+
+  "english":"English",
+  "german":"German",
+  "french":"French",
+  "japanese":"Japanese",
+
+  "options.language.h3":"Languages",
```

Comment:
ok so after going through this a few time I found what seems 'off' : translations are for UI text strings, the whole 'options' thing  seems like a sort of json in string and does not seems odd.
Can you explain your reasoning please ?